api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/loader/request/LoaderRangedFeeDto.java

```diff
@@ -24,7 +24,7 @@
     @JsonProperty("range_unit")
     private String rangeUnit;
 
-    public LoaderRangedFeeDto(String code, String newCode, LoaderFeeVersionDto version, String jurisdiction1, String jurisdiction2, String service, String channel, String event, String applicantType, BigDecimal maxRange, BigDecimal minRange, String keyword) {
-        super(code, newCode, version, jurisdiction1, jurisdiction2, service, channel, event, applicantType, false, keyword);
+    public LoaderRangedFeeDto(String code, String newCode, LoaderFeeVersionDto version, String jurisdiction1, String jurisdiction2, String service, String channel, String event, String applicantType, BigDecimal maxRange, BigDecimal minRange, String keyword, String reasonForUpdate) {
+        super(code, newCode, version, jurisdiction1, jurisdiction2, service, channel, event, applicantType, false, keyword, reasonForUpdate);
```

Comment:
What is the requirement to add 'reasonForUpdate' in as attribute in FeeDTO Model ?

Reply:
This also updated but forgot to push, did that along with Encode